src/mitsubishi2mqtt/mitsubishi2mqtt.ino

```diff
@@ -1794,7 +1794,7 @@ void loop() {
 		//MQTT failed retry to connect
 		if (mqtt_client.state() < MQTT_CONNECTED)
 		{
-		  if ((millis() > (lastMqttRetry + MQTT_RETRY_INTERVAL_MS)) or lastMqttRetry == 0) {
+		  if ((millis() - lastMqttRetry > MQTT_RETRY_INTERVAL_MS) or lastMqttRetry == 0) {
```

Comment:
I don't have write access to this repo but this looks right to me. When `millis()` wraps around,  `millis() < lastMqttRetry` and then `millis() - lastMqttRetry` should be a very large unsigned value, always greater than `MQTT_RETRY_INTERVAL_MS`. 

Now that I think about it, I've probably also been bitten by this bug!